eth/contracts/PancakeStableSwap/IPancakeStableSwapLP.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+interface IPancakeStableSwapLP {
```

Comment:
What about an API that returns the current swap ratio? It's liquidity pool API or another?